src/main/java/io/aiven/kafka/connect/s3/config/S3SinkConfig.java

```diff
@@ -215,6 +219,18 @@ private static void addAwsConfigGroup(final ConfigDef configDef) {
             AWS_SECRET_ACCESS_KEY_CONFIG
         );
 
+        configDef.define(
+                AWS_CREDENTIAL_PROVIDER_CONFIG,
+                Type.CLASS,
+                AWS_CREDENTIAL_PROVIDER_DEFAULT,
+                Importance.MEDIUM,
+                "AWS Credential Provider",
```

Comment:
Let's add some more detailed description here on when it's used and what defaults are used as well.

Reply:
Addressed.